src/main/java/frc/robot/hardware/gyro/EmptyGyro.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.hardware.gyro;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.EmptyDevice;
+
+public class EmptyGyro extends EmptyDevice implements IGyro {
+
+	public EmptyGyro(String logPath) {
+		super(logPath);
+	}
```

Comment:
I don't think you need to implement this

Reply:
you do